spec/rubocop/cop/lint/detect_invalid_pack_directives_spec.rb

```diff
@@ -0,0 +1,850 @@
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10].pack('V')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10].pack('U')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10].pack('w')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('D')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('d')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('F')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('f')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('E')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('e')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('G')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        [10.10].pack('g')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('A')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('a')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('Z')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('B')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('b')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('H')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('h')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('u')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('M')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('m')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('p')
+      RUBY
+    end
+
+    it 'does not raise an offence' do
+      expect_no_offenses(<<~RUBY)
+        ['test'].pack('P')
+      RUBY
+    end
+  end
+
+  context 'when the pack/unpack method contains an known directive across multiple lines' do
```

Comment:
I think this context needs updated in some way but I'm not sure what it was originally meant to mean 👀